packages/admin/src/components/LoginForm/index.tsx

```diff
@@ -0,0 +1,101 @@
+import { useState, useEffect } from "react";
+import { cx } from "@emotion/css";
+import { useForm } from "react-hook-form";
+import { AiOutlineLock, AiOutlineUnlock } from "react-icons/ai";
+import { IoPersonOutline } from "react-icons/io5";
+import getStyle from "./style";
+
+interface FormProps {
+  inputId: string;
+  inputPw: string;
+}
+
+export default function LoginForm() {
+  const [message, setMessage] = useState("");
+  const [classNameId, setClassNameId] = useState(false);
+  const [classNamePw, setClassNamePw] = useState(false);
+  const [lock, setLock] = useState(true);
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm();
+  const style = getStyle();
+
+  const onSubmit = (data: FormProps) => {
+    console.log(data);
+  };
+
+  const id = register("id", { required: true });
+  const password = register("password", { required: true });
+
+  useEffect(() => {
+    if (errors.id) setMessage("아이디를 입력해주세요.");
+    else if (errors.password) setMessage("비밀번호를 입력해주세요.");
+    else setMessage("");
+    console.log(errors);
+  }, [errors.id, errors.password]);
+
+  return (
+    <main className={cx(style.loginForm)}>
+      <span className={style.title}>CMI 관리자</span>
+      <form className={style.form} onSubmit={handleSubmit(onSubmit)}>
+        <div
+          className={
+            classNameId ? `${style.idBox} ${style.focus}` : `${style.idBox}`
+          }
```

Comment:
`style.idBox`가 공통이고 `classNameId` 값에 따라서 ` ${style.focus}`가 변하는거라면 `cx`를 활용하면 좋을것 같아요
```suggestion
          className={
            cx(style. idBox,{style.focus: classNameId})
          }
```
그리고 변수명도 `isFocusId` 이런식으로 변경하면 좋을것 같습니다

Reply:
오호 자세한 피드백 감사합니다~